jetty-server/src/test/java/org/eclipse/jetty/server/HttpOutputTest.java

```diff
@@ -54,6 +58,8 @@
  */
 public class HttpOutputTest
 {
+    public static final int OUTPUT_AGGREGATION_SIZE = 1024;
+    public static final int OUTPUT_BUFFER_SIZE = 4096;
```

Comment:
The test should parametrize on these values, something like `[512, 512], [512, 1024], [1024, 1024], [1024, 4096]`.

Reply:
why? what does the absolute size of the buffer have to do with it?